src/fat.rs

```diff
@@ -1172,16 +1185,16 @@ mod tests {
 
     #[test]
     fn test_name_to_str() {
-        let mut s = [0_u8; 11];
+        let mut s = [0_u8; 12];
         super::name_to_str("X       ABC", &mut s);
         assert_eq!(crate::common::ascii_strip(&s), "X.ABC");
-        let mut s = [0_u8; 11];
+        let mut s = [0_u8; 12];
         super::name_to_str(".", &mut s);
         assert_eq!(crate::common::ascii_strip(&s), ".");
-        let mut s = [0_u8; 11];
+        let mut s = [0_u8; 12];
         super::name_to_str("..", &mut s);
         assert_eq!(crate::common::ascii_strip(&s), "..");
-        let mut s = [0_u8; 11];
+        let mut s = [0_u8; 12];
         super::name_to_str("ABCDEFGHIJK", &mut s);
         assert_eq!(crate::common::ascii_strip(&s), "ABCDEFGHIJK");
```

Comment:
```suggestion
        assert_eq!(crate::common::ascii_strip(&s), "ABCDEFGH.IJK");
```
Should we need to fix the test case?